CHANGES

```diff
@@ -1,3 +1,13 @@
+0.15.0 (trunk):
+* Name.domain_name has been renamed to Name.t and is now abstract
+* Name.domain_name_to_string has been renamed to Name.to_string
```

Comment:
worth keeping domain_name_to_string as an alias to keep libres3 working for a while?
https://travis-ci.org/mirage/ocaml-dns/jobs/56403232#L2134